### PR TITLE
chore: move init to generators

### PR DESCRIPTION
**Pull request by rishabh3112**
**What kind of change does this PR introduce?**
chore/refactor

**Did you add tests for your changes?**
N/A

**If relevant, did you update the documentation?**
Will need to deprecate `init` package afterwards

**Summary**
Moved `init` command to generator pacakage

**Does this PR introduce a breaking change?**
No

**Other information**
Part of roadmap

### CODE REVIEW

File: packages/webpack-cli/package.json [hunk 1]
```diff
@@ -29,7 +29,6 @@
   "dependencies": {
     "@discoveryjs/json-ext": "^0.5.0",
     "@webpack-cli/configtest": "^1.0.0",
-    "@webpack-cli/info": "^1.2.1",
```
Comment (alexander-akait) [inline packages/webpack-cli/package.json:32]: No need to touch it

Comment (rishabh3112) [reply packages/webpack-cli/package.json:32]: Oops, thought it to be `init`. reverting.

**Review by jamesgeorge007 — COMMENTED** [review]
Please remove references to the `init` package from docs.

**Comment by rishabh3112** [comment]
> Please remove references to the `init` package from docs.

Deprecation will be done after this PR is public released. Deleting will lead to inaccessiblilty to people using current public release as we point them to master branch docs.

File: test/version/version.test.js [hunk 2]
```diff
@@ -157,17 +156,6 @@ describe('single version flag', () => {
...
-    it('outputs version with init', () => {
-        const { exitCode, stderr, stdout } = run(__dirname, ['init', '--version'], false);
-
-        expect(exitCode).toBe(0);
-        expect(stderr).toBeFalsy();
-        expect(stdout).toContain(`@webpack-cli/init ${initPkgJSON.version}`);
-        expect(stdout).toContain(`webpack-cli ${pkgJSON.version}`);
-        expect(stdout).toContain(`webpack ${webpack.version}`);
-        expect(stdout).toContain(`webpack-dev-server ${webpackDevServerPkgJSON.version}`);
-    });
```
Comment (alexander-akait) [inline test/version/version.test.js:169]: Why it was deleted?

Comment (rishabh3112) [reply test/version/version.test.js:169]: As init is deleted as of this PR, this tests version of init.

Comment (snitin315) [reply test/version/version.test.js:169]: So `webpack init --version` isn't supported anymore?

Comment (alexander-akait) [reply test/version/version.test.js:169]: Should work, we just move `init` to generators package, `webpack init --version` should still work

Comment (rishabh3112) [reply test/version/version.test.js:169]: Hmm, My bad. It should be same as generators' version now. Will add both tests with updates.

**Review by alexander-akait — COMMENTED** [review]
Please remove `packages/init/lib` from `.eslintignore` and `.prettierignore`

**Review by snitin315 — APPROVED** [review]
LGTM

**Review by jamesgeorge007 — APPROVED** [review]

**Comment by alexander-akait** [comment]
@rishabh3112 Can you rebase?

**Comment by rishabh3112** [comment]
> @rishabh3112 Can you rebase?

Will do asap, sorry for wait.

**Review by alexander-akait — APPROVED** [review]

**Review by snitin315 — APPROVED** [review]

**Comment by alexander-akait** [comment]
Thanks

**Pull request merged**